flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/sql/SqlITCase.scala

```diff
@@ -686,6 +686,28 @@ class SqlITCase extends StreamingWithStateTestBase {
 
     assertEquals(List(expected.toString()), StreamITCase.testResults.sorted)
   }
+
+  @Test
+  def testNullableFunctionCall(): Unit = {
```

Comment:
We should test this more lightweight with a unit test instead of running a complete query.
We can extend `ScalarFunctionsTest.testLPad()` with 

```
testSqlApi(
   "LPAD('hello', -1, 'x') IS NULL", "true"
)
```

to cover this case